ledger/participant-state/kvutils/src/test/suite/scala/com/daml/ledger/participant/state/kvutils/OffsetBuilderSpec.scala

```diff
@@ -4,46 +4,73 @@
 package com.daml.ledger.participant.state.kvutils
 
 import com.daml.ledger.offset.Offset
-import com.daml.lf.data
+import com.daml.lf.data.Bytes
 import org.scalatest.matchers.should.Matchers
 import org.scalatest.wordspec.AnyWordSpec
+import org.scalatestplus.scalacheck.ScalaCheckDrivenPropertyChecks
```

Comment:
How much slower does it make the test?

Reply:
On my machine the test file runs in ~100ms.

Reply:
How long did it take before?